conf/routes

```diff
@@ -2,86 +2,86 @@
 # This file defines all application routes (Higher priority routes first)
 # ~~~~
 
-GET         /robots.txt                      @controllers.CachedAssets.at(path="/public", file="robots.txt")
-GET         /favicon.ico                     @controllers.CachedAssets.busted(path="images/favicons/32x32.ico")
+GET         /robots.txt                      controllers.CachedAssets.at(path="/public", file="robots.txt")
```

Comment:
the @ sign means it uses the traditional way to create the controller, but now we switch to the compile time DI so I removed it

Reply:
👍